windows/src/engine/keyman32/keyman-debug-etw.man

```diff
@@ -0,0 +1,340 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<instrumentationManifest
+    xmlns="http://schemas.microsoft.com/win/2004/08/events"
+    xmlns:win="http://manifests.microsoft.com/win/2004/08/windows/events"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    >
+
+  <instrumentation>
+    <events>
+
+      <!-- resourceFileName and messageFileName must be set at install time; we'll use a common
+           default to make it easy for developers to install it from the source
+      -->
+      
+      <provider name="Keyman-Debug-ETWProvider"
+          guid="{DA621615-E08B-4283-918E-D2502D3757AE}"
+          symbol="ProviderGuid"
+          resourceFileName="%CommonProgramFiles(x86)%\keyman\keyman engine\keyman32.dll"
+          messageFileName="%CommonProgramFiles(x86)%\keyman\keyman engine\keyman32.dll"
```

Comment:
I'm not finding `keyman32.dll` in my local install `C:\Program Files (x86)\Keyman\...`
Is this something that gets installed with this PR?

Reply:
1. That's `C:\Program Files (x86)\Common Files\Keyman\...` because it's `%CommonProgramFiles%`.
2. It's not important anyway because those fields are replaced during the install of Keyman -- this is useful only for developers doing debug with the manifest (probably only me...)